core/src/boot.py

```diff
@@ -1,21 +1,47 @@
+import utime
```

Comment:
i would prefer to import and call `show_welcome_screen` directly from trezorui2 right here, to have it on screen before we start importing things. imports take a relatively long time so showing the screen before that would be nice.

we should skip fadein/fadeout, the screen is simple enough that it should not be a problem?

(we should also remove the fadein/fadeout sequence from later in this file, so that it does not hide the welcome screen)

Reply:
RIght, tried it and it is shown really very quickly.

Currently I have it like:

```python

# isort:skip_file

import utime
import trezorui2
from trezor import ui

# Showing welcome screen as soon as possible.
# Remembering time to control how long we show it.
trezorui2.show_welcome_screen().paint()  # type: ignore [Cannot access member "paint" for type "object"]
welcome_screen_start_ms = utime.ticks_ms()

# Setting up the display light.
# (For some reason needs to be done _after_ painting the welcome screen)
ui.display.backlight(ui.BACKLIGHT_NORMAL)

import storage
...
```

- it is strange that the backlight needs to be called after drawing

Reply:
what happens if you put it before drawing? :)

Reply:
The screen is just blank, without showing anything - and after that 1500 ms it shows the green homescreen normally
![image](https://user-images.githubusercontent.com/42543243/220664684-ff5ff9d6-dfaf-41fd-a2ca-a7f1d3b56629.png)

Reply:
i see, that might be an emulator-specific problem possibly?

in this case it would be great if you could also do the backlight call from Rust, because then you can postpone importing `trezor.ui`

Reply:
it's most likely a matter of not calling `display.refresh()` which the emulator requires and the real device is less stringent about.
that too should be done in Rust, after the paint call: sync -> paint -> refresh